lib/data_migrate/config.rb

```diff
@@ -12,12 +12,21 @@ def config
   end
 
   class Config
-    attr_accessor :data_migrations_path, :db_configuration, :spec_name
+    attr_accessor :data_migrations_path, :data_template_path, :db_configuration, :spec_name
+
+    DEFAULT_DATA_TEMPLATE_PATH = "data_migration.rb"
 
     def initialize
       @data_migrations_path = "db/data/"
+      @data_template_path = DEFAULT_DATA_TEMPLATE_PATH
       @db_configuration = nil
       @spec_name = nil
     end
+
+    def data_template_path=(value)
+      @data_template_path = value.tap do |path|
+        raise ArgumentError, "File not found: '#{path}'" unless path == DEFAULT_DATA_TEMPLATE_PATH || File.exists?(path)
```

Comment:
Performing validation on the setter seems a bit unorthodox, alternatively this can be performed from within the data migration generator i.e. https://github.com/ilyakatz/data-migrate/pull/232/files#diff-a4575045875fb9b1c551bd2d20696bd405d5c84ed26c2c773a895ce660c392ceR30

Reply:
i like the idea of doing validation here so that the error is raised on startup and devs will know right away that something is wrong. so looks good to me